components/brave_welcome_ui/components/select-browser/style.ts

```diff
@@ -62,7 +62,7 @@ export const BrowserListBox = styled.div`
     justify-content: center;
     align-items: center;
     flex-direction: column;
-    color: ${(p) => p.theme.color.text01};
+    color: #212529;
```

Comment:
Why are we changing some of these SC themed props to css variables and some to direct values?

Reply:
The text color on this item should be consistent regardless of the theme. A hardcoded value makes sense.

The border color on the item changes based on if the user has selected or not. A css variable here makes sense.

I'm not completely sure what you mean here.

Reply:
In some places in this PR, you have changed from SC theme prop (e.g. `p.theme.color.interactive05`) to css theme variable (e.g. `var(--interactive5)`) and in others you have changed from SC theme prop (e.g. `p.theme.color.text01`) to the value (e.g. `#212529`). I was wondering what the purpose of that inconsistency is...

Reply:
`var(--interactive5)` is used to decorate the checkboxes. I saw this being available in root and utilized it. The hardcoded value `#212529` wasn't.

Reply:
I'm still not sure why sometimes we're using `{(p) => p.theme.color.interactive08` (and not `--interactive8` from app.global.scss), and sometimes we're using `--interactive5` but not `p.theme.color.interactive05` and that you've made a conscious decision to change some and not others 🤷  

If you don't want the colors to ever change in dark / light then it's best to not use any of the theme variables.

Reply:
How did you resolve this?